streams/shared/src/main/scala/scalaz/stream/ZSink.scala

```diff
@@ -344,21 +382,45 @@ trait ZSink[-R, +E, +A0, -A, +B] { self =>
       def extract(state: State) = self.extract(state)
     }
 
-  final def dropWhile[A1 <: A](pred: A1 => Boolean): ZSink[R, E, A0, A1, B] =
-    new ZSink[R, E, A0, A1, B] {
-      type State = (self.State, Boolean)
+  /**
+   * Creates a sink that drops all produced elements on the floor.
```

Comment:
😂quite the literal description...
```suggestion
   * Creates a sink that ignores all produced elements.
```